app/script/user/UserRepository.js

```diff
@@ -1065,7 +1065,7 @@ z.user.UserRepository = class UserRepository {
   }
 
   map_guest_status(user_ets = this.users()) {
-    const team_members = this.team_members();
+    const team_members = this.teamMembers();
```

Comment:
maybe the `const` as well?

Reply:
Will happen when I refactor /users

Reply:
right..

Reply:
just saw that it's in the `/user` makes sense..